### ISSUE TITLE
[MSTAGE-7] Unable to use archiva as http source with maven-stage-plugin

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-stage-plugin#25]
**[Dan Tran](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dantran)** opened **[MSTAGE-7](https://issues.apache.org/jira/browse/MSTAGE-7?redirect=false)** and commented

http-wagon-beta-2 currently not able to parse the archiva page ( http://jira.codehaus.org/browse/MRM-891 ).

however with the fix from http://jira.codehaus.org/browse/MRM-893 and upgrade to wagon-beta-3 will solve the problem
without change in stage plugin source code.

---

**Affects:** 1.0-alpha-1

**Attachments:**
- [MSTAGE-7.patch](https://issues.apache.org/jira/secure/attachment/12719585/MSTAGE-7.patch) (_2.71 kB_)

1 votes, 0 watchers

**Comment by jira-importer** [comment]
**[Jacob Robertson](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=jacob%20robertson)** commented

I'm pretty sure this issue applies to the latest version of Nexus as well.  We upgraded our version of nexus, and the stage plugin stopped working.  The behavior was a never ending console logging of "../ found".  I applied this patch locally to the stage plugin, and the problem went away.
